Makefile

```diff
@@ -0,0 +1,52 @@
+.PHONY: all
+
+# Used for GDAL installation
+export CPLUS_INCLUDE_PATH=/usr/include/gdal
+export C_INCLUDE_PATH=/usr/include/gdal
+export PIP_INDEX_URL=https://pypi.pacificclimate.org/simple
+
+# Makefile Vars
+SHELL:=/bin/bash
+VENV?=~/tmp/ceb-venv
```

Comment:
I still think that we should use the system's temp directory (so $TMPDIR if set, else /tmp), but it can't be random (i.e. we can't use the `mktemp` command because it has to persist from one run to the next).

Maybe we can make it:

${TMPDIR:-/tmp}/ceb-venv

Not sure if that will work exactly *in* the Makefile, but that's how you express it in `bash`.

Reply:
Its not quite as pretty, but I did manage to set this up.